### PR TITLE
Added support for exporting chart image without rendering the chart first

### PR DESCRIPTION
**Pull request by ThumbGen**
Ref #451

### CODE REVIEW

File: Plotly.Blazor/PlotlyJsInterop.cs [hunk 1]
```diff
@@ -1,348 +1,367 @@
...
+    /// <summary>
+    ///     Can be used to export the chart as a static image and returns a binary string of the exported image.
+    /// </summary>
+    /// <param name="chartData">The chart data to be exported.</param>
+    /// <param name="format">Format of the image.</param>
+    /// <param name="height">Height of the image.</param>
+    /// <param name="width">Width of the image.</param>
+    /// <param name="cancellationToken">CancellationToken</param>
+    /// <returns>Binary string of the exported image.</returns>
+    public async Task<string> ToImage(object chartData, ImageFormat format, uint height, uint width, CancellationToken cancellationToken)
```
Comment (sean-mcl) [inline Plotly.Blazor/PlotlyJsInterop.cs:359]: Can you maybe use the same logic like in `Relayout<T>` for serialization of an anonymous type.

Comment (sean-mcl) [reply Plotly.Blazor/PlotlyJsInterop.cs:359]: So people dont need "dynamics" :)

File: Plotly.Blazor/wwwroot/plotly-interop-5.2.0.js [hunk 2]
```diff
@@ -43,6 +43,14 @@ function onPlotlyReady(divId, callback) {
     }
 }
 
+function onPlotlyReadyFromChartData(callback) {
```
Comment (sean-mcl) [inline Plotly.Blazor/wwwroot/plotly-interop-5.2.0.js:46]: Is that really required?  onPlotlyReady should be sufficient.

**Review by sean-mcl — CHANGES_REQUESTED** [review]

File: Plotly.Blazor/wwwroot/plotly-interop-5.2.0.js (hunk 2 above)
Comment (ThumbGen) [reply Plotly.Blazor/wwwroot/plotly-interop-5.2.0.js:46]: Yes, sadly checking for the div id breaks the logic. In the new case there is no div id. I had no better idea than duplicating the function :(

Comment (ThumbGen) [reply Plotly.Blazor/wwwroot/plotly-interop-5.2.0.js:46]: Ah, maybe passing a null divId could be handled in the original function and the check skipped.

Comment (ThumbGen) [reply Plotly.Blazor/wwwroot/plotly-interop-5.2.0.js]: Fixed

**Comment by ThumbGen** [comment]
It would be awesome if we could get this change packed in a new version; we already adjusted our code to 'welcome' it :)

**Pull request merged**